### DISCUSSION TITLE
Chatbot for Microsoft Teams

### DISCUSSION THREAD

**Discussion by trda** [FlowiseAI/Flowise#534] [Q&A]
I want to create a chatbot for Microsoft Teams. It will read the text written in the Microsoft Teams chat channel and respond using ChatGPT. I already have a tool to send messages to Teams, but I'm wondering how I can read the incoming messages from Microsoft Teams and pass them to ChatGPT.

**Comment by chungyau97** [comment]
Hi @trda, 

You can achieve that easily using [Zapier Zaps](https://zapier.com/app/dashboard) 

Implementation [docs](https://docs.flowiseai.com/3rd-party-platform-integration/zapier-zaps)

**Comment by gamersalpha** [comment]
i m looking for the same things, for now, i found nothing :/
